<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(197, 255, 187)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="WeatherTable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>961</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QChartView" name="Diagram">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>591</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>121</width>
      <height>941</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background  :  rgb(249, 255, 183)

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FillButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Fill table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddDayButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Add new day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AverageHumButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Average
humidity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Max 
temperature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinPressureButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Min pressure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WindChangeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Wind no 
changes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AverageHumidity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="WindChanges">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MinPressure">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MaxTemperature">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>860</y>
      <width>921</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PeriodButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Temperature
 by period</string>
    </property>
   </widget>
   <widget class="QListWidget" name="InfoPeriod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>770</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Last">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>810</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="First">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>770</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePeriodButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Period of 
change</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ChangePeriod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DiagramButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Diagram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Clear table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForecastButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Weather
forecast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>770</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string> - first index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>810</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string> - last index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(26, 26, 25);
border-radius : 15;</string>
    </property>
    <property name="text">
     <string>Informations</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
